armi/physics/thermalHydraulics/settings.py

```diff
@@ -27,7 +27,16 @@ def defineSettings():
             CONF_DO_TH,
             default=False,
             label="Run Thermal Hydraulics",
-            description="Run thermal hydraulics",
+            description=(
+                f"Activate thermal hydraulics calculations using the physics module defined in "
+                f"`{CONF_TH_KERNEL}`"
+            ),
+        ),
+        setting.Setting(
+            CONF_TH_KERNEL,
+            default=False,
```

Comment:
Should this have options, default None?

Reply:
or default="", rather? Whatever appeases the settings validation. `neutronicsKernel` uses options=[] and default="".